packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -105,6 +105,9 @@ export interface VercelServerlessConfig {
 
 	/** The maximum duration (in seconds) that Serverless Functions can run before timing out. See the [Vercel documentation](https://vercel.com/docs/functions/serverless-functions/runtimes#maxduration) for the default and maximum limit for your account plan. */
 	maxDuration?: number;
+
+	/** Whether to allow the serverless function to stream the response to the browser. */
+	streaming?: boolean;
```

Comment:
I feel like @sarah11918 might have thoughts on the wording here!

Reply:
I'm hoping lol

Reply:
I want to see which direction the README takes before bringing that over here! Don't let me forget about this, though!